tyr-core/pom.xml

```diff
@@ -58,12 +58,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
-      <version>${version.com.fasterxml.jackson}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${version.com.fasterxml.jackson}</version>
```

Comment:
where else is this needed? I see you moved it to the parent.

Reply:
Nowhere, deleted.